bpdm-gate-client/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/client/config/GateClientServiceConfig.kt

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.eclipse.tractusx.bpdm.gate.client.config
+
+import org.eclipse.tractusx.bpdm.gate.client.service.GateClientAddressInterface
+import org.eclipse.tractusx.bpdm.gate.client.service.GateClientBusinessPartnerInterface
+import org.eclipse.tractusx.bpdm.gate.client.service.GateClientLegalEntityInterface
+import org.eclipse.tractusx.bpdm.gate.client.service.GateClientSiteInterface
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.support.WebClientAdapter
+import org.springframework.web.service.invoker.HttpServiceProxyFactory
+
+class GateClientServiceConfig(
```

Comment:
I don't see this as a configuration but the actual client logic for the gate. Better name it SimpleGateClient or GateClientImpl